packages/adblocker/src/engine/engine.ts

```diff
@@ -271,6 +274,9 @@ export default class FilterEngine extends EventEmitter<
     }
     engine.lists = lists;
 
+    // Deserialize preprocessors
+    engine.preprocessors = PreprocessorBucket.deserialize(buffer);
```

Comment:
I wonder if it could make sense to make the `engine.preprocessors` optional (also while serializing). This way, if the preprocessors are disabled, the serialized engine will still be backward-compatible (i.e. we can load an existing serialized engine).

Reply:
would it still work with https://github.com/ghostery/adblocker/pull/3782/files#diff-a8978de7a0196a875248e88d7774100d4455d8cb7ddff861eb6903d747a00abdR89 ?

Reply:
Yes, this is not related.